catalog/controller/custom/api.php

```diff
@@ -316,14 +310,13 @@ private function getOrderTotals($order_id, $shippingCode, $countryID)
      */
     private function getTotalsTaxRate($key, $countryID)
     {
-        $DBPREFIX = DB_PREFIX;
-        $query = $this->db->query("select {$DBPREFIX}tax_rate.* from {$DBPREFIX}tax_rate
-              inner join {$DBPREFIX}setting on {$DBPREFIX}setting.`key` = '{$key}_tax_class_id'
-                INNER JOIN {$DBPREFIX}tax_class on {$DBPREFIX}tax_class.tax_class_id = {$DBPREFIX}setting.value
-              INNER JOIN {$DBPREFIX}zone_to_geo_zone on {$DBPREFIX}zone_to_geo_zone.country_id = {$countryID}
-              left JOIN {$DBPREFIX}tax_rule on {$DBPREFIX}tax_rule.tax_class_id =  {$DBPREFIX}tax_class.tax_class_id
-            where {$DBPREFIX}tax_rate.geo_zone_id = {$DBPREFIX}zone_to_geo_zone.geo_zone_id and {$DBPREFIX}tax_rule.tax_rate_id = {$DBPREFIX}tax_rate.tax_rate_id
-            group by {$DBPREFIX}tax_rate.tax_rate_id");
+        $query = $this->db->query("select {$this->DbPrefix}tax_rate.* from {$this->DbPrefix}tax_rate
```

Comment:
@byldrm $this->dbPrefixi çok kullanmışız. Kısa bir değişken mi yapsak, $prefix gibi fonksiyonun hemen başında.